docs/feature_flags.md

```diff
@@ -40,7 +40,17 @@ with more recent data.
 
 More details can be found [here](querying/basics.md#offset-modifier).
 
-## Remote Write Receiver
+# Evaluation-time-aligned subqueries in PromQL
+
+`--enable-feature=promql-eval-aligned-subqueries`
+
+As opposed to the normal `[x:y]` subquery syntax, which aligns with multiples of the subquery's
```

Comment:
Consider making this consistent with the [documentation](https://prometheus.io/docs/prometheus/latest/querying/basics/#subquery), which uses different terms:

```
 '[' <range> ':' [<resolution>] ']'
```